src/components/ui/common/Topology.js

```diff
@@ -34,7 +34,9 @@ exports.getNode = function( nodeName ){
 };
 
 exports.clickNode = function( nodeName ){
-    return exports.getNode(nodeName).then(function(node){
-        return node.click();
+
+    exports.getNode(nodeName).then(function(node) {
+        node.click();
     });
+    return browser.sleep(1000);
```

Comment:
click suppose to return a promise that would fulfil when the click is done.
returning a browser sleep remove this (maybe unused?) feature ,also is not consistent and we should not slow the system tests manually.

Maybe we should use the return promise to run our code instead of just waiting a second?